go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v1.0.0
 	github.com/golang-jwt/jwt/v4 v4.0.0
-	github.com/google/go-cmp v0.4.0
+	github.com/google/go-cmp v0.5.7
```

Comment:
If the update is not really needed I'll updated it in a different PR.

Reply:
ok back to the old version